zero-module.yml

```diff
@@ -196,6 +196,16 @@ parameters:
     - action: KeyMatchCondition
       matchField: billingEnabled
       whenValue: "yes"
+  - field: cacheStore
+    label: "Whether to use cacheStore"
+    info: "Options of cache store include Redis/Memcached"
+    options:
+      "none": "none"
+      "memcached": "Memcached"
+      "redis": "Redis"
+  - field: serverless
+    label: "Use Lightweight Serverless setup instead of Kubernetes Cluster SaSS setup"
```

Comment:
Will need to fix this label and add some improved info, maybe with a link out to a documentation page about the differences, upgrade process, etc.